cypress/e2e/2048-expert/user-action.cy.js

```diff
@@ -0,0 +1,81 @@
+const WIDTH = 6
```

Comment:
`input.cy.js` might be a better name as this is what Liedle uses to describe the test file for handling user input

Reply:
sure, I will change the name to make our program consistent